tests/test1.cpp

```diff
@@ -86,14 +88,10 @@ namespace Test1 {
       // add some lines
       //plot.add_object(*Gtk::manage(new Gtk::PLplot::PlotObject2DLine(-5, -1, 30, -2)));
       plot.add_object(*Gtk::manage(new Gtk::PLplot::PlotObject2DLine(30, -2, -5, -1)));
-      plot.add_object(*Gtk::manage(new Gtk::PLplot::PlotObject2DLine(Gtk::ORIENTATION_HORIZONTAL, -1)));
-      plot.add_object(*Gtk::manage(new Gtk::PLplot::PlotObject2DLine(Gtk::ORIENTATION_VERTICAL, 12)));
-
+      plot.add_object(*Gtk::manage(new Gtk::PLplot::PlotObject2DLine(Gtk::Orientation::HORIZONTAL, -1)));
+      plot.add_object(*Gtk::manage(new Gtk::PLplot::PlotObject2DLine(Gtk::Orientation::VERTICAL, 12)));
 
       set_default_size(720, 580);
-      Gdk::Geometry geometry;
```

Comment:
Is there an equivalent API in gtkmm4 to ensure the aspect ratio is fixed? If so, I would prefer that this geometry constraint is kept in all tests